autonomi/src/client/mod.rs

```diff
@@ -256,7 +239,8 @@ impl Client {
         Ok(Self {
             network,
             client_event_sender: Arc::new(None),
-            evm_network: config.evm_network,
+            evm_network: Arc::new(config.evm_network),
+            operation_config: Arc::new(config.operation_config),
```

Comment:
Why do we need `Arc` here?

Reply:
Every component Network is arc'ed to be freely clonable without incurring much cost, so I assume we'd want to do the same for Clients.